packages/jaeger-ui/src/components/TracePage/TraceJSON/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useEffect, useState } from "react";
+
+const TraceJSON = ({ trace, raw }: any) => {
```

Comment:
Is this related to this PR? Looks like new functionality, can we split it?

Reply:
This is part of the comment about the dropdown entries "Trace Json" and "Trace Json (unadjusted)" where you suggested to add a button to go back when the user clicks on one of the entries.  I just didn't know how to solve the issue with the json window because it was automatically opened and outside of react. I was not able to add a button, so I added a new page where the json data is displayed. If necessary, it can be splitted, yes.

Reply:
I would like to keep the PR focused on one thing - in this case preventing opening new windows.

I did not mean that we need to implement a Back button - it already exists in the browsers. Just showing the JSON in the current window is enough.

Reply:
Unfortunately, there is no back button. The Jaeger UI is embedded inside a global area surrounded with a notification area that provides access to other apps. I can do it in a way that the JSON will be opened in the current window. So we would remove this change from this PR and then I create a new PR with this solution?

Reply:
Here is an image of one of the instruments:

![98_Roche_Cobas_6800_system_01](https://user-images.githubusercontent.com/45211351/233995522-81250da6-d636-455f-9d20-5931859bb227.jpg)

Reply:
Do you need JSON view in your setup? The alternative would be to have an option to disable that functionality. If it's still required, let's do it in a separate PR. I don't have strong objections to rendering JSON ourselves, the main concern is that when traces are very large, rendering JSON manually can choke up the browser, whereas using browser-native view for JSON may still work ok. Browsers also tend to have extensions for viewing JSON with extra features like syntax coloring, expand/collapse, search, etc.- not the things we want to start adding to Jaeger as a custom JSON view.

Reply:
Thanks for the response. Then I will add a new option in the configuration to deactivate the json view because it is not needed (as far as I know).